src/modules/reservation.js

```diff
@@ -0,0 +1,126 @@
+import { displayReservations } from './displayReservations.js';
+// eslint-disable-next-line no-inner-declarations
```

Comment:
Let's not disable linters, whenever linters highlight errors, it's our job to correct those errors.